src/content/docs/iast/troubleshoot-iast.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Troubleshooting for interactive applications security testing (IAST)
+tags:
+  - IAST
+  - Exploitable vulnerabilities
+  - Troubleshooting
+metaDescription: Use Vulnerability Management to find exploitable vulnerabilities in your application.
+---
+
+## I don't see my application in New Relic
+
+Confirm that the application has started and then review the application's logs for further information.
+
+## How do I make sure that the security agent is working?
+
+In the application's `stdout` you will see this message:
+
+```shell
+This application instance is now being scanned by New Relic Security under id {{UUID}}
+```
+
+In the `nr-security-home/logs` directory, search for the file called `java-security-collector-init.log`, replace `java` in the path depending on language you used, and make sure these steps work for you:
+
+1. The security agent is starting.
+2. It generates a unique identifier. For web socket connection, you'll see Node auth headers.
+3. The agent gathers information about your application.
+4. The web socket connection to SaaS validator is established successfully.
+5. The security agent threads are started.
+6. The application instrumentation is successful.
+7. The application receives and applies your policies and configuration.
+8. You see a first event sent for validation, which means the security agent started successfully.
```

Comment:
For all of these steps, I think we need a little more clarity about where I can verify all of these things. I get the feeling that some things are in an application console and some things are in the New Relic UI, but it's tough for me to split them out without knowing more.

Reply:
Same as above, this is all part of the log file. It'll be easier to understand with the file excerpt.